### PR TITLE
fix actionType

### PR DESCRIPTION
**Pull request by AlvaroVega**
fix for https://github.com/telefonicaid/perseo-fe/issues/391

### CODE REVIEW

File: lib/models/updateAction.js [hunk 1]
```diff
@@ -447,9 +447,10 @@ function doRequestV2(action, event, token, callback) {
         var changes = processOptionParams(action, event);
         metrics.IncMetrics(event.service, event.subservice, metrics.actionEntityUpdate);
         logger.debug('entity to update: %j', changes);
-        // FIXME: createEntity with upsert true is always updating entity, but is not using params.actionType (append or update)
-        // Maybe other methods like v2.updateEntityAttributes is more proper ?
-        connection.v2.createEntity(changes, { upsert: true }).then(
+        var updateOptions = {
+            upsert: action.parameters.actionType === 'append' ? true : false
+        };
+        connection.v2.createEntity(changes, updateOptions).then(
```
Comment (fgalan) [inline lib/models/updateAction.js]: Is this the same logic other actionTypes appart from "append" and "update"? I mean, "delete" action, for deleting entities.

Comment (AlvaroVega) [reply lib/models/updateAction.js]: According with perseo doc:
```
actionType: optional, type of CB action: APPEND or UPDATE. By default is APPEND.
```
only append or update are expected.
Is posible actionType=delete for a POST /v2/enttities ?

Another cuestions could be if actionType is always in lowercase or not.

Comment (fgalan) [reply lib/models/updateAction.js]: Possible actionTypes in NGSIv2 are: "update", "append", "appendStrict", "delete" and "replace". There are two possible strategies:

* Use a library method adapted to the particular case. For instance, delete should use deleteEnity(), etc.
* Use the batch operation without looking to the actionType, in a transparent way from the point of view of Perseo. Maybe this is the easiest (although with a bit of overhead using a batch operation for an operation in a single entity.

Comment (AlvaroVega) [reply lib/models/updateAction.js]: Actualizado en https://github.com/telefonicaid/perseo-fe/pull/392

**Pull request merged**